seccomp_test.go

```diff
@@ -559,6 +559,28 @@ func TestMergeFilters(t *testing.T) {
 	}
 }
 
+func TestAddRuleErrors(t *testing.T) {
+	execInSubprocess(t, subprocessAddRuleErrors)
+}
+
+func subprocessAddRuleErrors(t *testing.T) {
+	filter, err := NewFilter(ActAllow)
+	if err != nil {
+		t.Errorf("Error creating filter: %s", err)
+	}
+	defer filter.Release()
+
+	err = filter.AddRule(ScmpSyscall(0x1), ActAllow)
+	if err == nil {
+		t.Error("expected error, got nil")
+	} else {
+		const exp = "matches default action"
```

Comment:
Alternatively, we can introduce
```go
var errRuleActionMatchesDefault = errors.New("requested action matches default action of filter")
```
in the code, and compare against it here in the test. This way, if the wording is ever changed, the test would not need fixing (but will still check the error returned is as expected).

Reply:
Implemented.